tests/www/approvals_views/__snapshots__/test_suspend.ambr

```diff
@@ -745,6 +822,83 @@
           LIMIT 1
         ''',
       }),
+      dict({
```

Comment:
vivement qu'on supprime les suspensions :roll_eyes: 

Il n'y a pas un prefetch que tu peux ajouter pour éviter 3 requêtes de plus ?
Je vois que `is_suspended()` est fait pour itérer en python sur `suspension_set` s'il est présent dans le cache
et `remainder()` utilise `self.suspension_set.all()` donc les 3 devraient a minima être fait dans une seule requête.
Pour la 3eme requête, ça donne un peu envie d'avoir un `approval.ongoing_suspension` qui utilise le même cache s'il est présent, mais vu qu'on va supprimer les suspension dans un avenir proche, c'est peut être inutile (enfin, sauf s'il faut gérer l'historique des suspensions pendant longtemps).

Reply:
J'ai rajouté un `prefetch_related` qui permet d'économiser 1 requête sur les 3 et je vais faire une autre PR pour l'`approval.ongoing_suspension` qui sera utile pour tous les affichages de l'encart :+1: